src/main/java/seedu/lifeasier/parser/Parser.java

```diff
@@ -67,6 +67,9 @@
 
     public static final int INDEX_START = 0;
     public static final int INDEX_END = 1;
+    public static final int MAXIMUM_CODE_LENGTH = 8;
+    public static final int MINIMUM_CODE_LENGTH = 6;
+    public static final int MAXIMUM_POSTFIX_CHARACTER = 1;
```

Comment:
I think you can consider renaming this to suffix!
```suggestion
    public static final int MAXIMUM_SUFFIX_CHARACTER = 1;
```

Reply:
Thanks for the tip, will change